src/analytics.ts

```diff
@@ -10,7 +10,7 @@ export type Geo = {
 export type Event = Geo & {
   identifier: string;
   time: number;
-  success: boolean;
+  success: boolean | string;
```

Comment:
Comment or Branded type highly appreciated.

Reply:
defined type EventSuccess. used "denied" instead of string. I initially thought using string would make it more flexible, but writing it in a restrictive way and extending later if needed makes more sense.